engines/python/setup/djl_python/request_output.py

```diff
@@ -0,0 +1,120 @@
+from dataclasses import field, dataclass
```

Comment:
this file contains both input and outputs, better use a difference file name

Reply:
Add license header

Reply:
Done